.storybook/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "rules": {
+    "import/no-extraneous-dependencies": [2, {
+      "devDependencies": true
+    }],
+    "react/jsx-filename-extension": 2,
+    "global-require": 2,
+    "comma-dangle": 2
```

Comment:
mostly to get my webpack config to stop yelling at me in sublime but also i guess maybe not necessary.

also not working. :/